src/core/reducers/data/details.ts

```diff
@@ -1,21 +1,17 @@
 import Actions from '../../actions';
 import Store from '../../store';
 
-export type Action = Actions.UpdateDetails | Actions.ReceiveDetailsProduct;
+export type Action = Actions.UpdateDetails;
 export type State = Store.Details;
 
 export const DEFAULTS: State = {};
 
 export default function updateDetails(state: State = DEFAULTS, action: Action): State {
   switch (action.type) {
     case Actions.UPDATE_DETAILS: return update(state, action.payload);
-    case Actions.RECEIVE_DETAILS_PRODUCT: return receiveProduct(state, action.payload);
     default: return state;
   }
 }
 
 export const update = (state: State, data: Store.Product) =>
   ({ ...state, data });
-
-export const receiveProduct = (state: State, product: Store.Product) =>
```

Comment:
Because you're removing this section, I think there's actually some cleanup you need to do surrounding the products section of details in the store, because I don't think it's ever used now. This means you also need to clean up UPDATE_PRODUCT_DETAILS action.

Reply:
Because gb-product expects an untransformed product so it can run productTransformer on it. If given an already transformed product it gives this: https://i.imgur.com/DCKnWJs.png

Reply:
Did you mean FETCH_PRODUCT_DETAILS? I already removed UPDATE_PRODUCT_DETAILS. The former is still needed, however, as when first loading a page you need to call that to get the full product dataset using the data in the url. It could be renamed to FETCH_DETAILS but I wanted to to avoid breaking things for clients. I've removed every superfluous action for product details however.

Reply:
mb wrong name `DETAILS_PRODUCT_UPDATED`